static_node_modules/opencascade.js/index.js

```diff
@@ -0,0 +1,13 @@
+import opencascade from "./dist/opencascade.wasm.js";
+import wasmFile from "./dist/opencascade.wasm.wasm";
+
+export const initOpenCascade = () => {
+  return new opencascade({
+    locateFile(path) {
+      if(path.endsWith('.wasm')) {
+        return wasmFile;
+      }
+      return path;
+    }
+  });
+}
```

Comment:
This was included in some of the other versions of opencascade.js, and it does make initializing the wasm file easier with webpack so I added back in.